src/arkode/arkode_arkstep_io.c

```diff
@@ -1479,6 +1480,8 @@ int ARKStepSetDefaults(void* arkode_mem)
 
 int ARKStepSetOptimalParams(void* arkode_mem)
 {
+  /* TODO: do we need to do something here? This is deprecated with no
+   * ARKodeSetOptimalParams to replace it */
```

Comment:
Is this deprecated? I though we ended up keeping these the "optimal" functions. @drreynolds